tldw_Server_API/tests/RAG_NEW/unit/test_pipeline_two_tier_gate.py

```
@@ -124,3 +124,53 @@ def _fake_create(strategy, cfg, llm_client=None):
             gate = res.metadata.get("generation_gate")
             assert isinstance(gate, dict)
             assert gate.get("reason") == "low_relevance_probability"
+
+
+@pytest.mark.asyncio
+async def test_two_tier_learned_fusion_abstention_policy_used(monkeypatch):
```

Ruff (0.14.4)

Unused function argument: `monkeypatch`

(ARG001)